packages/clay-core/src/table/Head.tsx

```diff
@@ -63,12 +62,15 @@ function HeadInner<T extends Record<string, any>>(
 
 	const collection = useCollection<T>({
 		children,
-		disabledKeys: new Set(hiddenColumns.keys()),
 		items,
 		suppressTextValueWarning: false,
+		visibleKeys: new Set(hiddenColumns.keys()),
```

Comment:
Hey @matuzalemsteles this looks problematic setting the value of `visibleKeys` to `hiddenColumns.keys()`. We need to use either visible or hidden throughout.

I think `hiddenKeys` makes more sense because we start with all keys as visible and then hide keys one by one. So, the default state is no keys are hidden which means `hiddenKeys` would be empty. When we hide one key, then `hiddenKeys` adds one item. 

If we use `visibleKeys` then the default value would need to include all keys, for it to make sense. That could work, but it still feels less intuitive than tracking which keys are hidden.

What do you think?

Reply:
By the way, the functionality is now working perfectly! 🎉

Reply:
Yes, I totally agree.